README.md

```diff
@@ -50,9 +50,13 @@ It's configured with auto connect IDE_KEY=phpstorm.
 Tests and metrics
 ======================
 
-To run all required test you can run the following command from the dev env:
+To run all required test you can run the following commands from the dev env:
 
-```composer test```
+``` 
+    ./bin/bootstrap_phantomjs.sh
+
+    composer test 
+```
```

Comment:
Running the tests failed in my dev environment. Behat failed with this message:

```
**> vendor/bin/behat  --config behat.yml
XDebug could not open the remote debug file '/tmp/xdebug.log'.
...........F-------......

--- Failed steps:

001 Scenario: When an user needs to register for a new token # src/AppBundle/Features/registration.feature:7
      Then I should see "Registration"                       # src/AppBundle/Features/registration.feature:11
        The text "Registration" was not found anywhere in the text of the current page. (Behat\Mink\Exception\ResponseTextException)

6 scenarios (5 passed, 1 failed)
25 steps (17 passed, 1 failed, 7 skipped)
0m8.24s (37.76Mb)
**
```

Reply:
Did you run vagrant provision from your machine? https is added recently